TABTestKit/Classes/Contexts/InteractionContext.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  InteractionContext.swift
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 09/09/2019.
+//
+
+import XCTest
+
+public protocol InteractionContext {} // TODO: AppContext
+public extension InteractionContext {
+	
+	func tap(_ element: Tappable) {
+		element.tap()
+	}
+	
+	func type(_ text: String, into element: Editable & Tappable, tappingFirst shouldTapFirst: Bool = true) {
```

Comment:
No issue on this line but I just wanted to what is this 'into; and Editable & Tappable wizardry? :O

Reply:
Both separate protocols, if you use & you can enforce that you can only pass in an element that conforms to both 🎉